serverless.yml

```diff
@@ -1,131 +1,94 @@
-service: my-store-app
-
-frameworkVersion: "3"
+service: comics-merch-frontend
+frameworkVersion: '3'
 
 provider:
   name: aws
-  runtime: nodejs14.x
-  region: eu-west-1
-  # setup profile for AWS CLI.
-  # profile: node-aws
+  runtime: nodejs16.x
+  region: ap-south-1
 
 plugins:
   - serverless-finch
-  # 'serverless-single-page-app-plugin' is a custom plugin that located .serverless_plugins folder.
-  # Existing plugin (https://www.npmjs.com/package/serverless-single-page-app-plugin) doesn't have invalidate cache feature that often used during CI/CD events.
-  # How to build your own plugins: https://www.serverless.com/framework/docs/providers/aws/guide/plugins#service-local-plugin
-  - serverless-single-page-app-plugin
+  # - serverless-single-page-app-plugin
```

Comment:
why do you comment  `serverless-single-page-app-plugin`?
According Evaluating Criteria we need to configure and add this plugin to **serverless** config. 
<img width="1238" alt="image" src="https://user-images.githubusercontent.com/110971794/219955516-5a1d5c3a-07b0-4880-a353-26c5cb97236e.png">

Reply:
I think I commented it by mistake while commiting the code. I deployed with uncommented code.